### PR TITLE
doc: DummySignInput mention external signer

### PR DESCRIPTION
**Pull request by Sjors**
Followups for #26032. So far nothing major.

### CODE REVIEW

File: src/wallet/wallet.cpp [hunk 1]
```diff
@@ -1648,8 +1648,8 @@ bool DummySignInput(const SigningProvider& provider, CTxIn &tx_in, const CTxOut
     const CScript& scriptPubKey = txout.scriptPubKey;
     SignatureData sigdata;
 
-    // Use max sig if watch only inputs were used or if this particular input is an external input
-    // to ensure a sufficient fee is attained for the requested feerate.
+    // Use max sig if watch only inputs were used, if this particular input is an external input,
```
Comment (glozow) [inline src/wallet/wallet.cpp:1651]: trailing whitespace here

**Comment by fanquake** [comment]
> Followups for https://github.com/bitcoin/bitcoin/pull/26032. So far nothing major.

Are you expecting there to be more? I really only see the one followup comment in #26032.

**Comment by Sjors** [comment]
Not really I guess...

**Comment by fanquake** [comment]
cc @S3RK @ishaanam @furszy

**Comment by ishaanam** [comment]
ACK 6fc5f4fdb661eb9d42842227501106afcf7111e7

**Comment by S3RK** [comment]
ACK 6fc5f4fdb661eb9d42842227501106afcf7111e7

The comment is correct, CI failure is unrelated

**Pull request merged**